lib/core/src/recover/recoverer.rs

```diff
@@ -280,9 +312,20 @@ impl Recoverer {
                 Swap::Chain(chain_swap) => match chain_swap.direction {
                     Direction::Incoming => {
                         let Some(recovered_data) = recovered_chain_receive_data.get(swap_id) else {
-                            log::warn!("Could not apply recovered data for incoming Chain swap {swap_id}: recovery data not found");
+                            warn!("Could not apply recovered data for incoming Chain swap {swap_id}: recovery data not found");
                             continue;
                         };
+
+                        let claim_is_cleared = chain_swap.claim_tx_id.is_some()
+                            && recovered_data.lbtc_claim_tx_id.is_none();
+                        if is_local_within_grace_period && claim_is_cleared {
```

Comment:
It could perhaps clear or reset (address reuse) the refund_tx_id here also

Reply:
Thanks. I was missing the btc part on both directions of chain swap. I've added that as well as the claim for outgoing chain swaps in [914ed5c](https://github.com/breez/breez-sdk-liquid/pull/733/commits/914ed5cf4f7a045c3a5be9a22469d3f732013a50).